plugin.video.reddit_viewer/addon.xml

```diff
@@ -2,7 +2,7 @@
 <addon
 	id="plugin.video.reddit_viewer"
 	name="Reddit Viewer"
-	version="3.3.8"
+	version="3.5.0"
 	provider-name="gedisony">
 	<requires>
 		<import addon="xbmc.python"                     version="2.24.0"                />
```

Comment:
JFYI: you can use the news tag to provide info for your users regarding the latest update. That will be shown on the addon website too. You're not required to add it though.